src/components/modules/shared/ArticleCountOptOut.tsx

```diff
@@ -69,16 +76,23 @@ const overlayContainer = (type: ArticleCountOptOutType): SerializedStyles => css
     }
 `;
 
+export interface OphanTracking {
+    componentType: OphanComponentType;
```

Comment:
Is `componentType` always `ACQUISITIONS_EPIC`? I'm not sure what this interface gives us

Reply:
At the moment that's the only one we're tacking, but the component is used in the contributions and subs banners too

Reply:
ah of course 👍